browse-building-floors/README.md

```diff
@@ -0,0 +1,39 @@
+# Browse building floors
+
+Display and browse through building floors from a floor-aware web map.
+
+![BrowseBuildingFloorsApp](browse-building-floors.png)
+
+## Use case
+
+Having map data to aid indoor navigation in buildings with multiple floors such as airports, museums, or offices can be incredibly useful. For example, you may wish to browse through all available floor maps for an office in order to find the location of an upcoming meeting in advance.
+
+## How to use the sample
+
+Use the spinner to browse different floor levels in the facility. Only the selected floor will be displayed.
+
+## How it works
+
+1. Create a `PortalItem` using the `itemId` of the floor-aware web map.
+2. Set the `MapView` to display the `PortalItem`.
+3. Wait for the map to load and retrieve the map's floor manager from `MapView.Map.FloorManager`. Then run `FloorManager.load()`.
+4. Wait for the floor manager to load using `FloorManager.addDoneLoadingListener` to retrieve the floor-aware data.
+5. Set all floors to not visible `FloorManager.levels[floor-number].isVisible = false`.
+6. Set only the selected floor to visible using `FloorManager.levels[floor-number].isVisible = true`.
+* **Note:** Manually set the default floor level to the first floor.
```

Comment:
This note is a bit ambiguous. Are you telling the users that it's not set by default?

Reply:
The could be a floor underneath the first floor (basement, parking, etc). By default sample display's the list of floors in the order by `FloorLevel.levelNumber`  which starts from `1` and cannot be negative. To display the first floor as the default floor the sample uses the floor with `FloorLevel.verticalOrder == 0`. The vertical order can be negative to represent floors below the first floor. 

This is explained more in the `MainActivity.kt` file. 
https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/blob/995fa921711475efeaecfdcf27f04165b6e92f25/browse-building-floors/src/main/java/com/esri/arcgismaps/sample/browsebuildingfloors/MainActivity.kt#L131-L139